ecs.tf

```diff
@@ -306,13 +311,14 @@ data "template_file" "worker" {
     rds_password                          = var.rds_password
     rds_db_name                           = local.rds_name
     airflow_core_logging_level            = var.airflow_core_logging_level
-    airflow_smtp_host                     = var.airflow_smtp_host
-    airflow_smtp_port                     = var.airflow_smtp_port
-    airflow_smtp_ssl                      = var.airflow_smtp_smtp_ssl
-    airflow_smtp_starttls                 = var.airflow_smtp_starttls
-    airflow_smtp_user                     = var.airflow_smtp_user
-    airflow_smtp_password                 = var.airflow_smtp_password
-    airflow_smtp_mail_from                = var.airflow_smtp_mail_from
+    airflow_smtp_smtp_host                = var.airflow_smtp_smtp_host
+    airflow_smtp_smtp_starttls            = var.airflow_smtp_smtp_starttls
+    airflow_smtp_smtp_ssl                 = var.airflow_smtp_smtp_smtp_ssl
+    airflow_smtp_smtp_port                = var.airflow_smtp_smtp_port
+    airflow_smtp_smtp_port                = var.airflow_smtp_smtp_port
```

Comment:
`airflow_smtp_smtp_port` seems to be duplicated

Reply:
oh, yeah! Better safe than sorry, ... will repair